docs/design/performance-benchmarking/WRITE-PERFORMANCE.md

```diff
@@ -0,0 +1,59 @@
+# Write Performance Benchmarking
+
+## Running Write Performance for Meadowlark
+
+There are two ways to analyze write performance for Meadowlark:
+
+1. Bulk Loading.
+2. Run Performance Testing Suite.
+
+## Bulk Loading
+
+> **Note** To run the bulk loading tests it's important to start with a clean
+> database.
+
+To load the data, there are functions to load the GrandBend and PartialGrandBend
+data sets into Meadowlark.
+
+To measure the execution time, run the script
+[Bulk-LoadPerformance.ps1](../../../eng/performance/BulkLoad-Performance.ps1).
+
+The script receives two parameters:
+
+- The **Template** that you desire to run (defaults to GrandBend).
+- The **Update** flag specifies if you desire to measure the creation or the
+  update of the resources (defaults to false).
+
+This script will enter the data into Meadowlark and will print the execution
+time.
+
+## Performance Testing Suite
+
+There are two performance test types that paths that cover the write
+performance, Pipeclean Tests and Volume Tests.
+
+### Setup
+
+The tests will be executed with
+[Suite3-Performance-Testing](https://github.com/Ed-Fi-Exchange-OSS/Suite-3-Performance-Testing)
+on branch [meadowlark-updates](https://github.com/Ed-Fi-Exchange-OSS/Suite-3-Performance-Testing/tree/meadowlark-updates) to have the changes required for Meadowlark.
+
+- Follow steps to [setup performance suite](./SETUP-PERFORMANCE-SUITE.md).
+- Go to /src/edfi_performance_test folder and run `poetry install`.
+- Create a user in meadowlark with the role of `assessment` (required to be able to insert the resources). Save key and secret.
```

Comment:
This shouldn't be necessary. Keep in mind that the ODS/API does not have a similar capability, and the test suite is working correctly with it.

Reply:
Maybe this is another issue that I should report, when using the admin key and secret from Meadowlark in the pipeclean tests and volume tests there are 0 inserts in the database, when using a user with the vendor role there are around 30% the amount of data inserted compared to a user with the assessment role

Reply:
You need to create a "vendor" key and secret without the additional "assessment" role. Admins are only allowed to create more credentials, not to manage resources.